examples/lightgbm_tuner_simple.py

```diff
@@ -0,0 +1,52 @@
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset using LightGBM tuner.
+
+In this example, we optimize the validation accuracy of cancer detection.
+We optimize both the choice of booster model and their hyperparameters.
```

Comment:
AFAIK, LightGBM Tuner doesn't optimize booster type, so this line seems somewhat misleading. Besides, I think that it's no problem to remove this line completely. What do you think of it?

Reply:
Yea, you're right :) I'll remove this sentence.